### PR TITLE
Discovery implementation and Kubernetes updates

### PR DESCRIPTION
**Pull request by tomlinton**
- Adds docker-compose setup for graphql and event listener
- Adds Kubernetes setup for graphql and event listener
- Updated all the Kubernetes containers to use remote branches rather than local repository for deployment now that the remote branches are in place
- Updated all the values files with current deployment state
- Misc fixes and improvements

Again sorry about the size @franckc!

### CODE REVIEW

File: development/scripts/create-es-index.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```
Comment (franckc) [inline development/scripts/create-es-index.sh]: Where is create-es-index.sh being used ? I did not see it referenced ?

File: docker-compose.yml [hunk 2]
```diff
@@ -64,6 +61,72 @@ services:
...
+      # dependency of the graphql server, but it is required here because the
+      # event-listener and origin-discovery share the same package.json
+      - ./origin-js:/usr/local/lib/node_modules/origin
+      - origin_js_node_modules:/usr/local/lib/node_modules/origin/node_modules
+    environment:
+      - DATABASE_HOST=postgres
+      - DATABASE_PORT=5432
+      - DATABASE_USERNAME=origin
+      - DATABASE_PASSWORD=origin
+      - DATABASE_NAME=origin-indexing
```
Comment (franckc) [inline docker-compose.yml]: I wonder if we should just call this database "origin" and use it not only for indexing but for everything ? Like in the future for storing mobile data, maybe for marketing purposes also (like tracking emails when people go thru attestation)...

File: development/scripts/create-es-index.sh (hunk 1 above)
Comment (tomlinton) [reply development/scripts/create-es-index.sh]: It's not yet, I didn't quite get to fully integrating it. I used it to manually create the indexes on the elasticsearch servers though.

File: docker-compose.yml (hunk 2 above)
Comment (tomlinton) [reply docker-compose.yml]: Makes sense. Not sure how we want to handle the situation where people want to deploy their own attestation server or indexing server though. The migrations would probably only create the required tables right? I was just thinking we don't want all the tables created.

**Pull request merged**